mlflow/server/js/src/experiment-tracking/components/CompareRunBox.js

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Row, Col, Select } from 'antd';
+import { Typography } from '@databricks/design-system';
+import { RunInfo } from '../sdk/MlflowMessages';
+import { LazyPlot } from './LazyPlot';
+import CompareRunUtil from './CompareRunUtil';
+
+const { Option, OptGroup } = Select;
+
+export const CompareRunBox = ({ runUuids, runInfos, metricLists, paramLists }) => {
+
+  const parameter = CompareRunUtil.getKeys(paramLists, true)[0];
+  const metric = CompareRunUtil.getKeys(metricLists, true)[0];
```

Comment:
What happens if the selected runs don't have any metrics and parameters?

Reply:
@harupy 
We need to implement a check on the paramter and metric to see if it is not Null or undefined. In the case of Null or undefined, the xAxis and yAxis should be set to undefined.

Reply:
Can we just set `xAxis` and `yAxis` undefined since we're not sure which parameters/metrics the user wants to plot?

Reply:
Ok sure!